### ISSUE TITLE
Question: are other types of fairness in scope?

### ISSUE THREAD

**Issue by adrinjalali** [fairlearn/fairlearn#244]
The terminology doc says:

> There are many approaches to conceptualizing fairness. In fairlearn, we follow the approach known as group fairness, which asks: Which groups of individuals are at risk for experiencing harms?

Does that mean other definitions of fairness are out of scope, or is that they're not included yet?

Also, are mitigation methods included here inherently incapable of working with other forms of fariness definitions?

Shouldn't it be up to the ethical counsel and lawyers and the local laws to define which types of fairness should be enforced? I'm not sure why this library should take a stand on that.

**Comment by romanlutz** [comment]
Anyone can add to this, but we try to focus on things that users request. Group fairness was what we started with, but if you want to add other kinds we'd love to have that as well!

The mitigation methods we have so far are for group fairness only.

We don't take a stand on what kind of fairness criteria is the right one. In fact, we quite explicitly say that fairness is a sociotechnical challenge, and every situation will require close examination by domain experts to figure out what makes sense. Group fairness is just one lens of assessing/mitigating unfairness, and often more tractable than individual fairness. That doesn't mean we want to discourage people from using other kinds. We explicitly encourage contributions of all kinds and I personally guarantee that I'll put in whatever time is necessary to get other techniques onboarded if anyone reaches out (at least unless we get flooded with such requests which I don't anticipate happening anytime soon). If you have ideas on what you'd like to add feel free to write a PR or set up some time to talk :-)

**Comment by adrinjalali** [comment]
That's awesome. I guess you can put that somehow in your "terminology" doc, since I got the question while reading that one.

**Comment by adrinjalali** [comment]
I if you wnat I could also try and send a PR with your explanation modifying the terminology doc.

**Comment by romanlutz** [comment]
Absolutely. I'm going to put out a PR with an update to markdown files (possibly README, but maybe also others) that shows clearly that we encourage participation and possible projects to get started with. This is something that needs to  be clarified along with that. Feel free to make some edits and send the PR :-)

**Comment by romanlutz** [comment]
#462 explains this a little bit (mentioning other types of fairness). @riedgar-ms FYI in case you want to elaborate on this more in your PR(?)

In general I think we're doing a better job of writing about this now than we did at the point the question came up.

**Comment by riedgar-ms** [comment]
I've tweaked the answer in the PR about this. Would welcome comments on the FAQ....

**Comment by hildeweerts** [comment]
Is this issue still relevant? @adrinjalali @romanlutz @riedgar-ms

**Comment by romanlutz** [comment]
I understand this as a question that turned into the realization that we should add/improve documentation including:
- what types of fairness are there (group vs. individual vs. ?)
- what do we have already and what can people add?

The existing documentation caused some confusion so we need to improve that.
